src/FormBuilder/resources/js/form-builder/src/index.tsx

```diff
@@ -33,7 +32,6 @@ setCategories([
 ]);
 
 registerHooks();
-registerComponents();
```

Comment:
We can't delete this at the moment, I did a quick search and can see fee recovery using `window.givewp.form.components.CurrencyControl`

<img width="1118" alt="Screenshot 2023-11-02 at 4 09 49 PM" src="https://github.com/impress-org/givewp/assets/10138447/c48c0419-7d23-4e1d-b641-5c81a8a6f381">

Reply:
Gooood catch! Reverted in 9401918